src/util/formattingNodeToHtml.ts

```diff
@@ -3,17 +3,41 @@ import _ from 'lodash'
 import stripStyleAttribute from './stripStyleAttribute'
 
 /** Strip span and encode a <i> or <b> element as HTML. */
+const replaceFontToSpan = (node: Element) => {
+  if (node.tagName === 'font') {
+    const newAttributes = node.attributes.map(attr => {
+      return attr.key === 'style'
+        ? { key: attr.key, value: attr.value }
+        : {
+            key: 'style',
+            value: `${attr.key}: ${attr.value}`,
+          }
+    })
+
+    // Return a new object that represents the updated node
+    return {
+      ...node,
+      tagName: 'span',
+      attributes: newAttributes,
+    }
+  }
+  return node
+}
+
+/** Converts a formatting node to HTML. */
 const formattingNodeToHtml = (node: Element) => {
   const content: string = node.children.reduce((acc, child) => {
     return acc + (child.type === 'text' ? child.content : child.type === 'comment' ? '' : formattingNodeToHtml(child))
   }, '')
-
+  node = replaceFontToSpan(node)
```

Comment:
Please create a new variable rather than mutating the old one. 

It's never a good idea to mutate arguments, as they are meant to be inputs only.